ngmix/priors.py

```diff
@@ -3017,36 +3029,36 @@ def get_lnprob_array2d(self, eta1, eta2):
     '''
 
 
-class Normal(object):
+class Normal(PriorBase):
     """
     This class provides an interface consistent with LogNormal
 
     C class defined get_lnprob_scalar(x) and get_prob_scalar(x)
     """
-    def __init__(self, cen, sigma, rng=None):
-        self.cen=cen
+    def __init__(self, mean, sigma, bounds=None, rng=None):
+        super(Normal,self).__init__(rng=rng, bounds=bounds)
+
+        self.mean=mean
```

Comment:
This might count as an API break, not that it totally matters.

Reply:
its optional.....

Reply:
Ohhhh I mean renaming `mean` to `cen` as an attribute.

Reply:
ahh, I see.  punt